src/com/content-event/event-idea.js

```diff
@@ -1,27 +1,26 @@
 import {h, Component} 				from 'preact/preact';
 import NavSpinner						from 'com/nav-spinner/spinner';
-import NavLink 							from 'com/nav-link/link';
 import SVGIcon 							from 'com/svg-icon/icon';
-
-import ButtonBase						from 'com/button-base/base';
-
+import UIButton							from 'com/ui/button/button';
 import $ThemeIdea						from '../../shrub/js/theme/theme_idea';
 
 
 const MAX_IDEAS = 3;
+const canHaveMoreIdeas = (ideas, submitting) => Object.keys(ideas).length + (submitting ? 1 : 0) < MAX_IDEAS;
+const hasSubmittedIdeas = (ideas) => Object.keys(ideas).length > 0;
```

Comment:
I don't have a nice way to suggest this via eslint, but here's my recommendation for how these lines should be formatted.

```js
const canHaveMoreIdeas = (ideas, submitting) => ((Object.keys(ideas).length + (submitting ? 1 : 0)) < MAX_IDEAS);
const hasSubmittedIdeas = (ideas) => (Object.keys(ideas).length > 0);
```

Yes functionally speaking they are the same, but the goal is to try and make the code easier to read with just a glance, by isolating elements with brackets. To keep things sane, the top-most level operation's brackets can be ignored. 

An example:

```js
// this is fine
if ( idea.length > 0 ) {
```
However:
```js
// but this
if ( idea.length > 0 && idea.length <= 64 ) {

// should be
if ( (idea.length > 0) && (idea.length <= 64) ) {
```

The top-level operation in this case is the boolean (`&&`) operations. In the above case, the top-level operation is the lambda (arrow function) operator.

There is an exception.

```js
// this is fine
if ( !this.thirsty && this.health ) {

// this is not
if ( !this.thirsty && this.health > 0 ) {

// it should be this
if ( !this.thirsty && (this.health > 0) ) {
```
So long as only a prefix/suffix operator is used, brackets can be ignored.

Reply:
I think there might be a lint rule for not mixing operators that could be used for these... perhaps.

Reply:
Sadly it's for mixing `&&` and `||` (and it's already enabled).